code-challenges.test.js

```diff
@@ -38,11 +71,30 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
+const onlyOdd = (arr) => {
+    const oddArray = arr.filter((oddArr) => typeof oddArr === "number" && oddArr % 2 !== 0)
+    return oddArray.sort((a, b) => a-b)
+}
+console.log(onlyOdd(fullArr1))
+console.log(onlyOdd(fullArr2))
+
+//This one was hard for me because I ran into the same issue I have a lot of the time, in where I know what to do, but not how to do it. i think i really need to work on understanding the syntax of things, because when I'm doing it i know I need to use the .filter higher order function, and to use typeof and all that, but putting everything together where it needs to go is difficult for me. I did also have to look up the syntax for the .sort function. 
+
```

Comment:
First question: 
Where is the pseudo code? 

Second question: 
What is it that you know how to do and can't figure out? How can I help? 

Third question: 
What are those console.logs doing?

Tip: don't name multiple variables the same thing. const oddArr and oddArray are NOT dealing with similar data - oddArr here is not an array but values in an array. mixedData would be a better name